### PR TITLE
Condense views table UI

### PR DESCRIPTION
**Pull request by paul121**
General improvements to the views table UI, exposed filters, and condensing space on pages

### CODE REVIEW

**Comment by mstenta** [comment]
Related discussion: https://farmos.discourse.group/t/ui-improvements-condensing-space/1732

File: modules/core/ui/views/config/install/views.view.farm_asset.yml [hunk 1]
```diff
@@ -1061,7 +1061,7 @@ display:
             image_target_id:
               align: ''
               separator: ''
-              empty_column: false
+              empty_column: true
```
Comment (mstenta) [inline modules/core/ui/views/config/install/views.view.farm_asset.yml:1064]: Do we want to set this on bundle-specific columns too? Currently this is only going to hide columns that are in the base View configuration, but we're also adding columns for bundle fields at runtime:

https://github.com/farmOS/farmOS/blob/399c01a177df14d4d2be547e28c6a98dee72fd63/modules/core/ui/views/farm_ui_views.views_execution.inc#L39

We aren't currently touching the `style.options.columns` configuration when we add columns in that hook, but maybe we should be - and without doing that it feels like this improvement is only half implemented. It wouldn't be obvious why some empty columns are hidden and others aren't.

File: modules/core/ui/theme/css/regions.css [hunk 2]
```diff
@@ -0,0 +1,23 @@
+/* Decrease the horizontal margin on medium to large screens. Gin uses xxl. */
```
Comment (mstenta) [inline modules/core/ui/theme/css/regions.css]: I wonder if we should split this out to another PR so they can be considered separately? As is they would be hidden under the "Views"-specific `CHANGELOG.md` line.

**Review by mstenta — CHANGES_REQUESTED** [review]

File: modules/core/ui/views/config/install/views.view.farm_asset.yml (hunk 1 above)
Comment (paul121) [reply modules/core/ui/views/config/install/views.view.farm_asset.yml:1064]: Agreed

File: modules/core/ui/theme/css/regions.css (hunk 2 above)
Comment (paul121) [reply modules/core/ui/theme/css/regions.css]: Yes - split to here https://github.com/farmOS/farmOS/pull/719

File: modules/core/ui/views/config/install/views.view.farm_asset.yml (hunk 1 above)
Comment (paul121) [reply modules/core/ui/views/config/install/views.view.farm_asset.yml:1064]: This was pretty easy! Also added a commit to let bundle fields be sortable.

**Review by mstenta — APPROVED** [review]

**Pull request merged**